packages/tests/esm/graphql-codegen.cjs

```diff
@@ -0,0 +1,34 @@
+const { mappers, enumValues } = require("./graphql-codegen-joist.cjs");
```

Comment:
I think ideally, we'd define this as a `.mjs` file (or `.ts` file) for consistency. I'll look into that.